int16 front_left_height
int16 front_right_height
int16 rear_left_height
int16 rear_right_height
time time_received